client/app/pages/alert/components/AlertDestinations.less

```diff
@@ -34,8 +36,8 @@
 
   .add-button {
       position: absolute;
-      right: 14px;
-      top: 9px;
+      right: 0;
+      top:-33px;
```

Comment:
This just locks the "+Add" button to its parent instead of the whole wrapper.
Needed cause this section gets pushed down by the "Notifications are muted" block.